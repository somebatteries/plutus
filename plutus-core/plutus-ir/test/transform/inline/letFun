(let
  (nonrec)

  (termbind
    (strict)
    (vardecl x (fun Bool Bool))
    ((lam x Bool x))
  )

  x
)
