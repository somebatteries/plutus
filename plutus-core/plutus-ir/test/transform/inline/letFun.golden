ParseErrorB (ParseErrorBundle {bundleErrors = TrivialError 79 (Just (Tokens ('(' :| "lam x "))) (fromList [Tokens ('a' :| "bs"),Tokens ('b' :| "uiltin"),Tokens ('c' :| "on"),Tokens ('e' :| "rror"),Tokens ('i' :| "wrap"),Tokens ('l' :| "am"),Tokens ('l' :| "et"),Tokens ('u' :| "nwrap")]) :| [], bundlePosState = PosState {pstateInput = "(let\n  (nonrec)\n\n  (termbind\n    (strict)\n    (vardecl x (fun Bool Bool))\n    ((lam x Bool x))\n  )\n\n  x\n)\n", pstateOffset = 0, pstateSourcePos = SourcePos {sourceName = "letFun", sourceLine = Pos 1, sourceColumn = Pos 1}, pstateTabWidth = Pos 8, pstateLinePrefix = ""}})